cloud_azure/run-cosmos-tests.sh

```
@@ -0,0 +1,213 @@
+
+# Find the system cacerts file
+JAVA_CACERTS=""
+if [ -n "$JAVA_HOME" ]; then
+    JAVA_CACERTS="$JAVA_HOME/lib/security/cacerts"
+elif command -v java >/dev/null 2>&1; then
+    # Try to find java home from the java command
+    JAVA_BIN=$(which java)
+    if [ -L "$JAVA_BIN" ]; then
+        JAVA_BIN=$(readlink -f "$JAVA_BIN" 2>/dev/null || readlink "$JAVA_BIN")
+    fi
+    POSSIBLE_JAVA_HOME=$(dirname $(dirname "$JAVA_BIN"))
```

Shellcheck (0.11.0)

[warning] Quote this to prevent word splitting.

(SC2046)